scripts/lib/platform-config.sh

```
@@ -67,22 +67,22 @@ validate_platform() {
 # Platform-specific environment setup
 setup_platform_env() {
     local platform="$1"
-    
+
     if ! validate_platform "$platform"; then
         return 1
     fi
-    
+
     # Set platform-specific environment variables
     export PLATFORM_SHELL=$(get_platform_config "$platform" "shell")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)